pubtools/_quay/tag_docker.py

```diff
@@ -666,7 +697,7 @@ def merge_manifest_lists_sign_images(
                     if outdated_manifests:
                         signer.remove_signatures(outdated_manifests, _exclude=current_signatures)
                     signer.sign_containers(
-                        to_sign_entries,
+                        to_sign_entries if signercls.pre_push else to_sign_entries_internal,
```

Comment:
It should always be to_sign_entries as it's already in if pre_push is true. And line #733 needs to be to_sign_entries_internal.

Reply:
fixed